core/src/test/java/nl/nn/adapterframework/jdbc/JdbcTransactionalStorageTest.java

```diff
@@ -200,12 +200,12 @@ private String createMessage() {
 
 	@Test
 	public void testRetrieveObjectWithADifferentColumnNotCompressed() throws Exception {
-		assertThrows("unknown compression method", JdbcException.class, () -> testRetrieveObjectWithADifferentColumnHelper(false));
+		assertThrows(JdbcException.class, () -> testRetrieveObjectWithADifferentColumnHelper(false));
 	}
 
 	@Test
 	public void testRetrieveObjectWithADifferentColumn() throws Exception {
-		assertThrows("invalid stream header", JdbcException.class, () -> testRetrieveObjectWithADifferentColumnHelper(true));
```

Comment:
Het is wel zo netjes de message dan als 3e argument mee te geven.

Reply:
dit is aangepast

Reply:
Dit is aangepast